server.js

```diff
@@ -170,6 +174,40 @@ async function createExpressApp() {
       })
   })
 
+
+expressApp.post('/createRoom', validateCreateRoomParams, async (req, res) => {
```

Comment:
To improve consistency with other API endpoints, we can put this 'createRoom'  as a POST to `/rooms/:roomId`, where `roomId` is optional and the other parameters (eventNotificationUri, maxPeerCount) are given as body parameters in JSON format.

Reply:
Resolved